OutOfSchool/OutOfSchool.WebApi/Services/AreaAdminService.cs

```diff
@@ -492,7 +479,7 @@ public async Task<bool> IsAreaAdminSubordinateRegionCreateAsync(string regionAdm
         return regionAdmin.InstitutionId == institutionId && regionAdmin.CATOTTGId == catottgId;
     }
 
-    private static Expression<Func<AreaAdmin, bool>> PredicateBuild(AreaAdminFilter filter)
+    private static Expression<Func<AreaAdmin, bool>> PredicateBuild(AreaAdminFilter filter, List<long> catottgs)
```

Comment:
(Open question) I see that some parts of filtering predicates are passed as `AreaAdminFilter` properties, but `catottgs` is passed as separate parameter. I can't check if adding this parameter as an `AreaAdminFilter` property will break other parts of the systme, but for isolated code I see here it looks like vital change. 

Please consider and redesign if it is valid.

Reply:
I see, in AreaAdminFilter all the properties are public and they can be changed and shown practically everywhere. I think there is no necessary to add a list with `catottgs` in this filter because it fills up just in `GetByFilter` method at once and nowhere else. Method `PredicateBuilder` was created mostly to decrease the amound of code in `GetByFilter` and has just 1 usage